### ISSUE TITLE
[Vert.x 4] gRPC Dev Mode tests failing with Vert.x 4.0.3

### ISSUE THREAD

**Issue by cescoffier** [quarkusio/quarkus#15753]
## Describe the bug

The dev mode tests are failing:

On the client side:

```
 io.grpc.StatusRuntimeException: UNAVAILABLE: Connection closing while performing protocol negotiation for [NettyClientHandler#0, WriteBufferingAndExceptionHandler#0, DefaultChannelPipeline$TailContext#0]
```

and on the server side:

```
2021-03-16 08:06:26,124 INFO [io.grp.net.Net.connections] (vert.x-eventloop-thread-0) Transport failed: io.netty.handler.codec.http2.Http2Exception: First received frame was not SETTINGS. Hex dump for first 5 bytes: 0000080700
at io.netty.handler.codec.http2.Http2Exception.connectionError(Http2Exception.java:108)
```

## To Reproduce

Just re-enabled the `io.quarkus.grpc.server.devmode.GrpcDevModeTest` (remove the `@Disabled` annotations), and run the tests.

I've disabled them for now.

**Comment by michalszynkiewicz** [comment]
is it the maven or the gradle one?

**Comment by cescoffier** [comment]
io.quarkus.grpc.server.devmode.GrpcDevModeTest is a maven one. Do I miss something?

**Comment by michalszynkiewicz** [comment]
fails for me too

**Comment by cescoffier** [comment]
Fixed by @michalszynkiewicz (#15874)
